common/src/main/resources/ts/models/FormElement/FormElement.ts

```diff
@@ -1,6 +1,25 @@
 import {Selectable} from "entcore-toolkit";
 import {FormElementType} from "@common/core/enums/form-element-type";
-import {FormElements, Question, Section, Types} from "@common/models";
+import {FormElements, Question, QuestionChoice, Section, Types} from "@common/models";
+
+export class FormElementIdType {
+    id: number;
+    type: FormElementType;
+
+    constructor(id: number, type: FormElementType) {
+        this.id = id ? id : null;
+        this.type = type ? type : null;
+    }
+
+    equals = (formElementIdType: FormElementIdType) : boolean => {
+        let areBothUndefined: boolean = this === formElementIdType === undefined;
```

Comment:
La première égalité renvoie un boolen qui ne peut jamais être égal à undefined donc ça renvoie toujours false non ?
let areBothUndefined = this === undefined && formElementIdType === undefined;

Reply:
Tout à fait, bien vu !